firebase-firestore/src/androidTest/java/com/google/firebase/firestore/ArrayTransformsTest.java

```diff
@@ -37,6 +38,9 @@
  */
 @RunWith(AndroidJUnit4.class)
 public class ArrayTransformsTest {
+
+  @Rule public RetryRule retryRule = new RetryRule(3);
```

Comment:
Can you add:

// TODO(b/114769487): These tests have been flaky in CI so temporarily disabling.